ts/components/Tooltip.tsx

```diff
@@ -5,6 +5,8 @@ import React from 'react';
 import classNames from 'classnames';
 import { noop } from 'lodash';
 import { Manager, Reference, Popper } from 'react-popper';
+import { StrictModifiers } from '@popperjs/core';
```

Comment:
Because you're only importing this as a type, could you change this to `import type`?

```suggestion
import type { StrictModifiers } from '@popperjs/core';
```